contracts/TokenManager.sol

```diff
@@ -320,9 +373,24 @@ contract TokenManager is ITokenController, IForwarder, AragonApp {
 
     function _mint(address _receiver, uint256 _amount) internal {
         require(_isBalanceIncreaseAllowed(_receiver, _amount), ERROR_BALANCE_INCREASE_NOT_ALLOWED);
+        for (uint256 i = 0; i < hooksLength; i++) {
+            if (address(hooks[i]) != 0) {
+                hooks[i].onTransfer(0x0, _receiver, _amount);
+            }
+        }
         token.generateTokens(_receiver, _amount); // minime.generateTokens() never returns false
     }
 
+    function _burn(address _holder, uint256 _amount) internal {
+        for (uint256 i = 0; i < hooksLength; i++) {
+            if (address(hooks[i]) != 0) {
+                hooks[i].onTransfer(_holder, 0x0, _amount);
+            }
+        }
```

Comment:
This is repeated in 2 places (3 if you remove `transferable` in one of them) and can be put in a separate function.

Reply:
I agree that the same code is repeated over and over. I guess I can do a `triggerHooks` function that loops over the hooks.